compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -531,11 +531,7 @@ void buildAMDAIETransformPassPipeline(OpPassManager &variantPassManager) {
         [&]() { return createAMDAIELowerExecutableTargetPass(options); });
   }
   modulePassManager.addPass(createLowerUKernelOpsToCallsPass());
-  if (clUsePipeline == AIEPassPipeline::PadPackPipeline) {
-    addMLIRAIRLoweringPasses(modulePassManager, false);
-  } else if (clUsePipeline == AIEPassPipeline::PackPeelPipeline) {
-    addMLIRAIRLoweringPasses(modulePassManager, true);
-  }
+  addMLIRAIRLoweringPasses(modulePassManager, clUsePipeline);
```

Comment:
I think you can simplify this function further. You don't have to pass `clUsePipeline` to this function.

Reply:
Make sense. Updated.